Taskfile.yml

```diff
@@ -101,35 +106,49 @@ tasks:
         task test -- tests/test_alignments.py
     deps: [poetry-install-tests]
     cmds:
+      - poetry install --only tests --only utils --no-root
       - PYTHONPATH=$(pwd) poetry run pytest -vv {{.CLI_ARGS}}
 
+  test-native:
```

Comment:
> Rename this to something like test-non-amd64

I don't think this is accurate, as the base docker image is multi-platform. It will use the "native" architecture for your OS, whether it is ARM, AMD64. So here native would be more accurate, while using `test-non-amd64` could be running on native amd64 system.

How about I make it more specific and use "multiplatform", as that's really the distinction here.

Reply:
Ah, apologies - I misunderstood what the naming here was for. I'm happy with the current version, or the new suggestion.

Reply:
I actually like the `multiplatform` name better, as it's more accurate as to what's going on.